odoo/addons/base/models/res_lang.py

```diff
@@ -265,11 +269,9 @@ def _lang_code_to_urlcode(self, code):
         return self._lang_get(code).url_code
 
     @api.model
-    @tools.ormcache()
     def get_installed(self):
         """ Return the installed languages as a list of (code, name) sorted by name. """
-        langs = self.with_context(active_test=True).search([])
-        return sorted([(lang.code, lang.name) for lang in langs], key=itemgetter(1))
+        return [(lg[0], lg[2]) for lg in self.get_available() if lg[3]]
```

Comment:
`get_available` is hooked by website and it will alter the result if website is in context (mainly for controllers/rpc), is that correct to use it here?